bitcoin/src/lib.rs

```diff
@@ -13,21 +13,28 @@
 //! See README.md for detailed documentation about development and supported
 //! environments.
 //!
-//! ## Available feature flags
+//! # Available feature flags
 //!
-//! * `std` - the usual dependency on `std` (default).
+//! * `default` - enables `std` and `secp-recovery`.
+//! * `std` - the usual dependency on `std`.
+//! * `serde` - implements `serde`-based serialization and deserialization.
+//! * `secp-lowmemory` - optimizations for low-memory devices.
 //! * `secp-recovery` - enables calculating public key from a signature and message.
+//! * `bitcoinconsensus` - (dependency) enables validation of scripts/transactions by `rust-bitcoinconsensus`.
```

Comment:
Why this is the only `(dependency)` that is not followed by a comma `,` like the others?
```suggestion
//! * `bitcoinconsensus` - (dependency), enables validation of scripts/transactions by `rust-bitcoinconsensus`.
```

Or remove the comma `,` from the others.